shared/src/clients/mail-client.class/index.ts

```diff
@@ -38,8 +38,11 @@ export default class MailClient {
     this.configSet = configSet
   }
 
-  public sendMail(input: MailToSend, option?: SendEmailOpts): Promise<boolean> {
-    return new Promise<boolean>((resolve, reject) => {
+  public sendMail(
+    input: MailToSend,
+    option?: SendEmailOpts
+  ): Promise<string | void> {
+    return new Promise<string | void>((resolve, reject) => {
```

Comment:
I think `Promise<string>` makes more sense, if the idea is it should be up to the call-site to decide if they don't want to use the `string` that's returned. if you look at the actual `resolve` clause, it always resolves as a string.

(The code below is actually not type-safe... but that's a battle for another time I guess)

```suggestion
  public sendMail(input: MailToSend, option?: SendEmailOpts): Promise<string> {
    return new Promise((resolve, reject) => {
```